.changeset/few-geckos-think.md

```diff
@@ -0,0 +1,7 @@
+---
+'@leafygreen-ui/popover': minor
+'@leafygreen-ui/portal': minor
+---
+
+Manually forward `_ref` to `Portal` component and expose `portalRef` in `Popover` component
```

Comment:
If we decide to update the prop name in the other PR, just be sure we update this here